README.md

```diff
@@ -1 +1,213 @@
-# in-the-air
\ No newline at end of file
+
+
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Space Travelers' Hub\] ](#-space-travelers-hub-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [`npm start`](#npm-start)
+    - [`npm test`](#npm-test)
+    - [`npm run build`](#npm-run-build)
+    - [`npm run eject`](#npm-run-eject)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [In The Air] <a name="about-project"></a>
+
+**"In The Air"** is working with the real live data from the open weather API.This application keeps track of air toxicity levels across the worlds' major cities.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>JSX</li>
+    <li>React Components</li>
+    <li>React Router V6</li>
+    <li>CSS</li>
+    <li>Redux Toolkit</li>
+    <li>Bootstrap</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built with React JS**
+- **Single page app**
+- **Redux store with reducers**
+- **Neat folder structure**
+- **Redux toolkit for state management**
+- **Neat user interface**
+- **Responsive design**
+- **Live demo**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+ ## 🚀 Live Demo <a name="live-demo"></a>
+- [Space Travelers Hub](https://in-the-air.onrender.com/)
+
+
```

Comment:
- [ ] You've done an outstanding job meeting the project requirements :+1:; however, including a video showcasing the project's features would be beneficial. This video should not exceed 5 minutes in length.
The addition of a video presentation would serve to provide a visual and more engaging overview of the project's key features, making it easier for stakeholders and viewers to understand and appreciate the project's capabilities and achievements :+1:.